src/components/Card/FAQcard.css

```diff
@@ -0,0 +1,27 @@
+.lil-faqcard {
+  /*
+  background-color needs to be updated
+  */
+  height: 280px;
+  width: 252px;
+  background-color: #1c314f;
+
+  @apply lil-rounded-xl lil-p-5  lil-cursor-pointer lil-text-center lil-m-2;
+}
+
+.lil-faqcard > .lil-faqcard-faq-wrapper {
+  @apply lil-mr-5 lil-w-full;
+}
+
+.lil-faqcard-faq-wrapper .lil-faqcard-faq {
+  height: 33%;
```

Comment:
can we use flex here and provide width based on utility exposed by tailwind

Reply:
used justify-between along with some margins